AEXS/aex-141.md

```diff
@@ -105,9 +109,21 @@ entrypoint metadata(token_id: int) : option(metadata)
 | data | option(metadata) |
 
 The `metadata` type to use depends on the `metadata_type` defined in the contract:
-- for `URL`, `IPFS` and `OBJECT_ID` use `MetadataIdentifier`
+- for `URL` and `OBJECT_ID` use `MetadataIdentifier`
 - for `MAP` use `MetadataMap`
 
+### total_supply\(\)
+
+Returns the total amount of NFTs in circulation.
+
+```sophia
+entrypoint total_supply() : ínt
+```
+
+| return | type |
+| :--- | :--- |
+| total_supply | int |
+
 ### balance\(\)
 
 Returns the account balance of another account with address `owner` if the account has a balance. If the owner address is unknown to the contract, `None` will be returned. Using `option` type as a return value allows us to determine if the account has balance of 0, more than 0, or the account has never had balance and is still unknown to the contract.
```

Comment:
balance should probably be better explained as "number of nfts owned in that collection"

Reply:
fixed: https://github.com/aeternity/AEXs/commit/e26c477d3069c36b489ab8a6cfcef94daf3b4e00